### DISCUSSION TITLE
Add a streamlit.code component similar to the streamlit module

### DISCUSSION THREAD

**Discussion by angel0614563** [zauberzeug/nicegui#1536] [Ideas / Feature Requests]
### Description

I hope to add a st.code component similar to the one provided in the streamlit module in the official document of Nicegui, displaying different code blocks and having a one click copy function.

**Comment by rodja** [comment]
I like the idea. Who want's to create a pull request? Inspiration can be taken from the code-window implementation on nicegui.io: 
https://github.com/zauberzeug/nicegui/blob/219da1118c30d0611ec665cb7cf8b185b36833de/website/demo.py#L68-L89

**Comment by falkoschindler** [comment]
This is a duplicate of #592.
But I'm not sure what to do about it, since both tickets already collected upvotes.
